### PR TITLE
@alloy => Use group flag to determine if shows with only one artist are actually group

### PR DESCRIPTION
**Pull request by mzikherman**

### CODE REVIEW

File: schema/show.js [hunk 1]
```diff
@@ -34,9 +34,11 @@ import {
   GraphQLUnionType,
 } from 'graphql';
 
-const kind = ({ artists, fair, artists_without_artworks }) => {
+const kind = ({ artists, fair, artists_without_artworks, group }) => {
   if (isExisty(fair)) return 'fair';
-  if (artists.length > 1 || (artists_without_artworks && artists_without_artworks.length > 1)) {
+  if (group ||
```
Comment (mzikherman) [inline schema/show.js]: I split it up cause the linter was complaining about the line length. Not sure this is better?

File: test/schema/show.js [hunk 2]
```diff
@@ -204,6 +204,26 @@ describe('Show type', () => {
           });
         });
     });
```
Comment (alloy) [inline test/schema/show.js]: Please add an empty line between scopes.

File: schema/show.js [hunk 3]
```diff
@@ -34,9 +34,11 @@ import {
...
+      artists.length > 1 ||
+      (artists_without_artworks && artists_without_artworks.length > 1)) {
```
Comment (alloy) [inline schema/show.js]: So Gravity returns a 'group' value but that is not implicitly derived from the number of artists in the show unless explicitly set?

File: schema/show.js (hunk 1 above)
Comment (alloy) [inline schema/show.js]: It's less pretty, but it's more helpful when ralearon the code on e.g. GitHub, which has a column width of I think 120.

Comment (mzikherman) [inline schema/show.js]: 👍

File: test/schema/show.js (hunk 2 above)
Comment (alloy) [inline test/schema/show.js]: Between each `it` block there should also be a newline.

File: schema/show.js (hunk 3 above)
Comment (alloy) [inline schema/show.js]: @mzikherman Following up from our offline chat, please remember to remove the redundant checks other than the use of the `group` field, seeing as Gravity already implicitly encodes those checks into `group`.

Comment (mzikherman) [inline schema/show.js]: 👍

**Pull request merged**